src/main/java/org/matsim/pt2matsim/gtfs/GtfsConverter.java

```diff
@@ -223,7 +223,7 @@ protected TransitLine createTransitLine(Route gtfsRoute) {
 	protected TransitRoute createTransitRoute(Trip trip, Map<Id<TransitStopFacility>, TransitStopFacility> stopFacilities) {
 		Id<RouteShape> shapeId = trip.getShape() != null ? trip.getShape().getId() : null;
 
-		if(trip.getStopTimes().size() == 0) {
+		if(trip.getStopTimes().size() >= 2) {
```

Comment:
Sorry, I didn't look at this properly. It should be `<= 1` since the method returns null if a trip is unusable. We want to keep going if there are more than 2 stops.

Reply:
yes, you are right... Didn't change it correctly. Sorry. I also meant <= 1

Reply:
Didn't we need at least 2 stops in order to calculate offsets

Reply:
Well, we need at least 2 stops in a trip to create a transit route. This if clause however returns with null from `createTransitRoute` if there are not enough stops (i.e. 0 or only 1 stop). The original clause only considered trips without stops, we want it to consider trips with 0 or 1 stop.

Reply:
I agree, so`<= 1` should be fine. Should I provide a 2nd pull request with this solution?

Reply:
Yes, it's cleaner that way. Thank you.